modules/p2/ding_interaction_manual/ding_interaction_manual.info

```diff
@@ -0,0 +1,22 @@
+name = Ding interaction manual
+description = Create custom ding interaction messages.
+package = Ding!
+version = "7.x-1.x-dev"
+core = 7.x
+dependencies[] = eck

```

Comment:
:+1: [It has now](https://github.com/ding2/ding2/pull/207/commits/d7d9d3d952b0927ffb9f6bc002bc4cef90cc0bad#diff-07b6005eb7dac5284919c0eb459c1d15R58).